src/net/sourceforge/kolmafia/session/EquipmentRequirement.java

```diff
@@ -0,0 +1,46 @@
+package net.sourceforge.kolmafia.session;
+
+import net.sourceforge.kolmafia.KoLConstants.Stat;
+import net.sourceforge.kolmafia.utilities.StringUtilities;
+
+public class EquipmentRequirement {
+  private Stat stat = Stat.NONE;
+  private int amount;
+
+  public EquipmentRequirement(String requirement) {
+    if (requirement.startsWith("Mus:")) {
+      stat = Stat.MUSCLE;
+      amount = StringUtilities.parseInt(requirement.substring(5));
+    }
+
+    if (requirement.startsWith("Mys:")) {
+      stat = Stat.MYSTICALITY;
+      amount = StringUtilities.parseInt(requirement.substring(5));
+    }
+
+    if (requirement.startsWith("Mox:")) {
+      stat = Stat.MOXIE;
+      amount = StringUtilities.parseInt(requirement.substring(5));
+    }
```

Comment:
Should we throw at the bottom here?

Reply:
no, it's possible for the requirement to be "none" (in which case stat is Stat.NONE and amount is 0)